pyvista/utilities/reader.py

```diff
@@ -647,6 +772,174 @@ class BinaryMarchingCubesReader(BaseReader):
     _class_reader = _vtk.vtkMCubesReader
 
 
+@dataclass(order=True)
+class PVDDataSet:
+    """Class for storing dataset info from PVD file."""
+
+    time: float
+    part: int
+    filename: str
+    group: str
+
+
+# skip pydocstyle D102 check since docstring is taken from TimeReader
+class PVDReader(BaseReader, TimeReader):
+    """PVD Reader for .pvd files.
+    
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> filename = examples.download_wavy(load=False)
+    >>> filename.split("/")[-1]  # omit the path
+    'wavy.pvd'
+    >>> reader = pyvista.get_reader(filename)
+    >>> reader.time_values  # doctest: +ELLIPSIS
+    [0.0, 1.0, 2.0, 3.0, ... 12.0, 13.0, 14.0]
+    >>> reader.set_active_time_point(5)
+    >>> reader.active_time_value
+    5.0
+    >>> mesh = reader.read()[0]  # MultiBlock mesh with only 1 block
+    >>> mesh.plot(scalars='z')
+
+    """
+
+    def __init__(self, filename):
+        """Initialize PVD file reader."""
+        self._reader = None
+        self.filename = filename
+        self._directory = None
+        self._datasets = []
+        self._active_datasets = []
+        self._active_readers = []
+        self._time_values = []
+
+        self._set_filename(filename)
+        
+    @property
+    def reader(self):
+        """Return the PVDReader.
+        
+        ..note::
+            This Reader does not have an uderlying vtk Reader.
+
+        """
+        return self
+
+    @property
+    def active_readers(self):
+        """Return the active readers.
+        
+        Returns
+        -------
+        list[:class:`pyvista.BaseReader`]
+
+        """
+        return self._active_readers
+
+    @property
+    def datasets(self):
+        """Return all datasets.
+        
+        Returns
+        -------
+        list[:class:`pyvista.PVDDataSet`]
+
+        """
+        return self._datasets
+
+    @property
+    def active_datasets(self):
+        """Return all active datasets.
+        
+        Returns
+        -------
+        list[:class:`pyvista.PVDDataSet`]
```

Comment:
Thanks to using `numpydoc`, we don't have to use the `class` directive anymore when writing returns.

```suggestion
        list[pyvista.PVDDataSet]
```